app/controllers/api/v0/base_controller.rb

```diff
@@ -2,6 +2,10 @@ class Api::V0::BaseController < ApplicationController
   include Swagger::Blocks
   include OpenStax::Swagger::Bind
 
+  rescue_from_unless_local StandardError, send_to_sentry: true do |ex|
+    render json: binding_error(status_code: 500, messages: [ex.message]), status: 500
+  end
```

Comment:
So this turns any exception into a 500 when sending to sentry. Are there other useful statuses we might want to use?

Reply:
@reedstrm there will be.  this is just a start.